src/engine/controls/loopingcontrol.cpp

```diff
@@ -18,6 +18,21 @@
 #include "track/track.h"
 #include "track/beats.h"
 
+namespace {
+inline mixxx::FrameDiff_t samplesToFrames(double samples) {
+    return samples / mixxx::kEngineChannelCount;
+}
+inline double framesToSamples(mixxx::FrameDiff_t frames) {
+    return frames * mixxx::kEngineChannelCount;
+}
+inline mixxx::FramePos samplePosToFramePos(double samplePos) {
+    return mixxx::FramePos(samplePos / mixxx::kEngineChannelCount);
+}
```

Comment:
This is exactly the same code as in QuantizeControl. Let's move it to a common file with a big warning that it should not be used for new code, only adapting legacy code to use frames.